docs/handoffs/codex-finalize/A10-hygiene.md

```
@@ -0,0 +1,220 @@
+| `sed -n '240,520p' docs/handoffs/2026-05-13-codex-finalize-distribution.md` | repo root | 0 |
+| `sed -n '1,240p' CLAUDE.md` | repo root | 0 |
+| `sed -n '1,220p' package.json` | repo root | 0 |
+| `sed -n '1,220p' tsconfig.json` | repo root | 0 |
+| `find code-oz-gui -maxdepth 2 -name 'package*.json' -o -name 'tsconfig*.json'` | repo root | 0 |
+| `sed -n '1,240p' code-oz-gui/package.json` | repo root | 0 |
+| `sed -n '1,220p' code-oz-gui/tsconfig.json` | repo root | 0 |
+| `sed -n '1,200p' code-oz-gui/.next/package.json` | repo root | 0 |
+| `sed -n '1,220p' code-oz-gui/README.md` | repo root | 0 |
+| `rg --files src tests code-oz-gui \| sed -n '1,220p'` | repo root | 0 |
+| `bun run typecheck` | repo root | 0 |
+| `git grep -nE 'TODO|FIXME|XXX'` | repo root | 0 |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 3; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)